libraries/AC_AttitudeControl/AC_AttitudeControl_Multi.cpp

```diff
@@ -74,6 +75,38 @@ const AP_Param::GroupInfo AC_AttitudeControl_Multi::var_info[] = {
     // @Increment: 0.5
     // @User: Advanced
 
+    // @Param: RAT_RLL_D_FF
+    // @DisplayName: Roll Derivative FeedForward Gain
+    // @Description: FF D Gain which produces an output that is proportional to the rate of change of the error
```

Comment:
are we missing the Range, Units and USer for RAT_RLL_D_FF?

Reply:
Thanks fixed as best I can